packages/server/src/api/rest/index.ts

```
@@ -1251,6 +1251,14 @@ class RequestHandler extends APIHandlerBase {
                     {}
                 ),
             };
+        } else {
+            return idFields.reduce(
+                (acc, curr, idx) => ({
+                    ...acc,
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)